multiversx_sdk_cli/cli_wallet.py

```diff
@@ -57,6 +75,12 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
         "new",
         "Create a new wallet and print its mnemonic; optionally save as password-protected JSON (recommended) or PEM (not recommended)",
     )
+    sub.add_argument(
+        "--validator-wallet",
```

Comment:
I wonder whether we should have a separate sub-command, instead.

E.g.

```
mxpy wallet ... (for user wallets)
mxpy validator-wallet ...
```

Especially since other sub-sub-commands of `mxpy wallet` do not make sense for this kind of wallets (e.g. _bech32_, _convert_ etc.). Actually, some _convert_ functionality could be useful (e.g. PEM to raw hex), but, overall, maybe we can have them separately.

:thought_balloon:

Reply:
Moved to a separate sub-command